payjoin-cli/Cargo.toml

```diff
@@ -28,9 +28,9 @@ v2 = ["payjoin/v2", "payjoin/io"]
 [dependencies]
 anyhow = "1.0.99"
 async-trait = "0.1.89"
+bitcoind-async-client = {git = "https://github.com/arminsabouri/bitcoind-async-client", rev = "956ca693e4263c003eaa4fa938d909d24acac5fa"}
```

Comment:
Guys you can use the 0.5.0 from crates.io: https://crates.io/crates/bitcoind-async-client

Cc @arminsabouri